templates/userdata.cfg.tpl.j2

```diff
@@ -7,3 +7,6 @@ packages:
   - git
   - vim
   - tmux
+ssh_authorized_keys:
+  - "{{ cluster_deploy_ssh_public_key }}"
+  - "{{ cluster_user_ssh_public_key }}"
```

Comment:
These won't be defined in manual deployments.

Reply:
It would be a bit cleaner to create a Terraform input for these, rather than templating twice.

Reply:
These are the SSH keys injected into the VM instance and shouldn't be needed for the manual deployment, therefore they should be set as empty during the manual deployment and ignored. This, again, is something that requires testing.